react/features/display-name/components/DisplayName.web.js

```diff
@@ -132,7 +132,7 @@ class DisplayName extends Component {
         if (allowEditing && this.state.isEditing) {
             return (
                 <input
-                    autoFocus = { true }
+                    autoFocus
```

Comment:
Hey @larspa! Thank you very much for the contribution!

Could I please ask you to remove the change of `autoFocus = { true }` to `autoFocus` from this PR? It's not consistent with how we write. Thank you!

Otherwise, great catch of the unnecessary string wrapping!